src/workingtitle-vcockpits-instruments/html_ui/Pages/VCockpit/Instruments/Shared/Map/WorkingTitle/MapInstrument.js

```diff
@@ -1215,44 +1217,18 @@ class MapInstrument extends ISvgMapRootElement {
     }
 	
     centerOnPlane() {
-        if (this.orientation == "north" || !this.offsetPlaneInHdgTrk) {
-			this.setNavMapCenter(this.navMap.planeCoordinates);
-			if (this.eBingMode == EBingMode.PLANE) {
-				this.airplaneIconElement.forceCoordinates(this.navMap.centerCoordinates.lat, this.navMap.centerCoordinates.long);
-			}
-		} else {
-			// MOD: if orientation is heading or track up, we want to place the plane 33% from the bottom of the map,
-			// but vector needs to be adjusted for overdraw factor of sqrt(2), which is where the magic number of 382 comes from (500-500/sqrt(2)+0.33*1000/sqrt(2))
-			let target = this.navMap.XYToCoordinatesFromPlaneWithRotation(new Vec2(500, 382));
-			this.setNavMapCenter(target);
-		}
+		// MOD: allow for arbitrary placement of plane
+		let r_x = this.navMap.aspectRatio < 1 ? this.navMap.aspectRatio : 1;
+		let r_y = this.navMap.aspectRatio > 1 ? 1 / this.navMap.aspectRatio : 1;
+		this.setNavMapCenter(this.navMap.XYToCoordinatesFromPlaneWithRotation(new Vec2(500 * (1 - r_x) + 1000 * (1 - this.planeTrackedPosX) * r_x, 500 * (1 - r_y) + 1000 * (1 - this.planeTrackedPosY) * r_y)));
     }
 	
     centerOnActiveWaypoint(_val) {
         this.bEnableCenterOnFplnWaypoint = _val;
     }
 	
-    // MOD: adapt this method to the new orientation model for compatibility purposes
+    // MOD: this method now does nothing
 	rotateWithPlane(_val) {
```

Comment:
What about a change in the default rotation handler so that it can return either 0 or HDG based upon the status of a flag set within it, but defaulting to 0.   Then we could update `rotateWithPlane` to toggle that flag in the default handler.   This would, if I'm thinking clearly, still let us keep backwards compatibility.   But I might be missing something.

Reply:
Yeah, I can definitely do something like this. How I do it will depend on whether we want to move all the orientation handling code back into MapInstrument per https://github.com/Working-Title-MSFS-Mods/fspackages/pull/206#discussion_r502108316

Reply:
After I left that comment I stepped away from the computer and realized that since we're using our own instrument backwards compatibility is a limited concern.  The only things using that are the 1000, 3000, and G3X, and we can easily adapt all those in one go.  Doing extra work for compatibility that we can easily never need to use doesn't make sense...   so I think I'm fine with leaving this as is.